Modules/Devices/src/Devices.Application/Clients/DTOs/ClientDTO.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Backbone.Modules.Devices.Application.Clients.DTOs;
+
+public class ClientDTO
+{
+    public ClientDTO(string? clientId, string? displayName)
+    {
+        ClientId = clientId;
+        DisplayName = displayName;
+    }
+    public string? ClientId { get; set; }
```

Comment:
The ClientId shouldn't be optional (same for the corresponding constructor parameter)